### DISCUSSION TITLE
Support for empty tables?

### DISCUSSION THREAD

**Discussion by jmesterh** [slingdata-io/sling-cli#114] [Ideas]
I don't see it in the documentation (maybe I'm missing it) but if I export all the tables using a wildcard, empty tables are skipped?

This is a problem for our users as their SQL statements will work on the source but not the target due to these missing tables.

**Comment by flarco** [comment]
Makes sense to create empty table with same structure. Will require a bit of tweaking to get it to work the way things are, not a priority. Creating Issue.

**Comment by flarco** [comment]
Shipped. See https://github.com/slingdata-io/sling-cli/issues/116.
